sources/osg/PrimitiveFunctor.js

```diff
@@ -370,4 +370,12 @@ PrimitiveFunctor.prototype = {
     }
 };
 
+var instance = new PrimitiveFunctor();
```

Comment:
It's not a singleton, it's just an instance.
You are free to create a new instance.

The reason to use a single instance is to avoid useless object creation, so no change in the api.

(if it were for me I'd simply make a function instead of a class, it doesn't store anything sensible, it's purely functional but I didn't want to introduce this kind of API change now)

Reply:
Well applyInstance instead of apply *is* an API change. I prefer also to have an static function than to change to this, as the API already changed.